### ISSUE TITLE
C++20 keywords

### ISSUE THREAD

**Issue by DannyParker0001** [djkaty/Il2CppInspector#106]
With C++20 some more keywords need to be added to the list.
'module', 'import', 'export' and 'requires'

**Comment by djkaty** [comment]
Is this up to date? https://en.cppreference.com/w/cpp/keyword

I don't see 'module' or 'import' on that list.

**Comment by djkaty** [comment]
I looked into this a bit further. As far as I can see all of the C++20 language keywords are already in the exclusion list:

https://github.com/djkaty/Il2CppInspector/blob/ccf56385d860aae12c02fdbf8a233394fa69c22f/Il2CppInspector.Common/Cpp/CppDeclarationGenerator.cs#L540

The words 'import' and 'module' together with the new meaning of 'export' are only recognized in the context of preprocessor directives which should not impact their use as type or field names. 'requires' is already on the exclusion list.

I'm going to close this issue, but if I have made a mistake feel free to comment and we can look again! 🙂
